docs/DeveloperGuide.md

```diff
@@ -180,11 +180,47 @@ is a new and undocumented `Author`, it is then added to `Tantou`'s `AuthorList`,
 of their manga authors. The `AuthorList` is saved via `Storage` for data persistency.
 #### Interaction
 The following diagram illustrates the interactions that take place when the
-user provides `catalog -a Kubo Tite` as an input.
-![add author sequence diagram](/docs/uml/images/addauthor.png)
+user provides `"catalog -a Kubo Tite"` as an input.
+![add author sequence diagram](/docs/uml/images/AddAuthorSequence.png)
```

Comment:
This diagram is well-documented, and is compliant with the course's understanding of sequence diagrams. Good job!